src/pages/home/ReportScreen.js

```diff
@@ -584,9 +599,7 @@ function ReportScreen({
                                         isEmptyChat={isEmptyChat}
                                         lastReportAction={lastReportAction}
                                     />
-                                ) : (
-                                    <ReportFooter isReportReadyForDisplay={false} />
-                                )}
+                                ) : null}
```

Comment:
What will this change accomplish? Will this mean that the Composer is not going to be visible when the report screen is loading?